tests/TestLiteDB.py

```
@@ -0,0 +1,32 @@
+        """Create temporary database for testing"""
+        fd, path = tempfile.mkstemp(suffix='.db')
+        os.close(fd)
+        db = LiteDB(path)
+        yield db
+        os.unlink(path)
+
+    def test_create_table(self, temp_db):
+        """Test table creation"""
+        fields = {"name": str, "age": int}
+        temp_db.create_table("test_table", fields)
+        tables = temp_db.show_tables()
```

Ruff (0.11.9)

Local variable `tables` is assigned to but never used

Remove assignment to unused variable `tables`

(F841)